res/app/components/stf/tokens/access-token-service.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function AccessTokenServiceFactory(
+  $rootScope
+, $http
+, socket
+) {
+  var AccessTokenService = {}
+
+  AccessTokenService.getAccessTokens = function() {
+    return $http.get('/app/api/v1/accessTokens')
```

Comment:
Just realised that calling api every time is making it a bit slow. Will keep it cached in user data (client side)